scriptshifter/tables/data/dogri_devanagari.yml

```diff
@@ -1,18 +1,19 @@
 ---
 general:
   name: Dogri (Devanagari)
-  case_sensitive: false
+  parents:
+    - _ignore_base
+    - _devanagari_base
 
-script_to_roman:
-  hooks:
-    post_config:
-      -
-        - aksharamukha.romanizer.s2r_post_config
-        - src_script: "Devanagari"
+  description: Dogri language in the Devanagari script
+  case_sensitive: false
 
 roman_to_script:
-  hooks:
-    post_config:
-      -
-        - aksharamukha.romanizer.r2s_post_config
-        - dest_script: "Devanagari"
+  map:
+    " " : " "
```

Comment:
@RandyBarry This effectively maps the space character, which is ignored in the `_ignore_base` config.

No harm done, it's just redundant and we can let it go for expediency's sake. 

To just have an empty map, you can also use `map: {}`

Or I can improve the code not to require `map` to be present and to be a dict.